backend/api/models/datamanager.py

```diff
@@ -24,11 +24,7 @@ class DataManager(models.Model, AssetPermissionMixin):
     owner = models.CharField(max_length=100)
     creation_date = models.DateTimeField()
     metadata = models.JSONField()
-    channel = models.CharField(max_length=100)
-
-    def get_data_samples(self):
```

Comment:
this function is no longer used?

Reply:
THis function was only used in the serializer